app/controllers/html_player_controller.rb

```diff
@@ -0,0 +1,10 @@
+class HtmlPlayerController < ApplicationController
+  include SlideUtil
+  before_action :set_slide
+  layout "simple"
```

Comment:
__[rubocop]__
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

<!-- Comment by [SideCI](https://sideci.com) -->